src/network/networkd-ndisc.c

```diff
@@ -187,6 +188,12 @@ static int ndisc_request_route(Route *in, Link *link, sd_ndisc_router *rt) {
                         return log_link_warning_errno(link, r, "Failed to get default router MTU from RA: %m");
         }
 
+       if (link->network->ipv6_accept_ra_use_hop_limit) {
```

Comment:
Avoid a configuration option when an RFC specifies the feature as a "SHOULD" requirement.